core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -1,9 +1,9 @@
 <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
   <h3>Topic Similarities</h3>
-  <div>
+  <div class="oppia-topic-Similarities-container">
```

Comment:
Why the inconsistent capitalization? Please fix.

Reply:
Done Fixed Thanks!